arcsi/handler/upload.py

```diff
@@ -251,8 +251,11 @@ def find_playlist_id(self,):
             for playlist in r.json():
                 if playlist["name"] == self.playlist_name:
                     self.playlist_id = str(playlist["id"])
-                    return True
+                    app.logger.debug("Add to playlist request returned {}".format(r.status_code))
```

Comment:
One small idea here would be (really just cosmetic, shouldn't keep you from merging) to do `request returned {:d}` to suggest clearly that the `r.status_code` is an `Integer`

Reply:
Good idea. Could you add it when doing this https://github.com/mmmnmnm/lahmacun_arcsi/issues/114?